### DISCUSSION TITLE
IaC周りを整理する

### DISCUSSION THREAD

**Discussion by jacopen** [cloudnativedaysjp/dreamkast#1387] [Ideas]
現状
- CFn (Dk周りのAWSリソース全般)
- CDK (Viewer Count)
- Terraform (AWSリソース以外)

という感じでIaCツールが共存している。
どこかでこのあたりを整理したい

**Comment by takaishi** [comment]
cdktf使ってみたいと思っている

**Comment by ShotaKitazawa** [comment]
個人的な意見ですが、自分はこの辺りのツールを制限する必要は無いと思っていて、以下さえ把握できる形であればある程度何を使っても良いとすべきと思っています。

* どこにデプロイされているのか
* デプロイに何が使われてるか
* デプロイ・削除する方法 (コマンド等)

というのも dreamkast 周りは特に「仕事で」というより「やってみたい」ベースで参加する人が多いと思っていて、その人がその時にやりたいものに着手できる方が全員にとって幸せなんじゃないかと思うためです。

またこれの懸念として明らかに筋が悪い方法を採用してしまうことが挙げられますが、この問題はSlack または GitHub issue 等で何を使うか事前 (または事後) に共有してもらうことで十分避けられる気がします。

**Comment by jacopen** [comment]
TF化していくのでこちらは閉じ！
